confbot_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.5)
+project(confbot_msgs)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(action_msgs REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  "action/MoveCommand.action"
+  DEPENDENCIES action_msgs
```

Comment:
Not: this should depend on `builtin_interfaces` I believe

Reply:
done in https://github.com/Karsten1987/roscon2018/pull/20/commits/dfcc14eef96f028b84631624be059cc8d416b0aa